tutorials/advanced_features/Virtual Devices.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<p>I would maybe not use DQC, just so it's not confused with the DQC protocol.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/451/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>